### PR TITLE
replace static with interactive map and markers

### LINKED ISSUE #166: [Frontend] Change HereMap from static image to interactive map
Current behavior:

Map component renders a static image with markers on indicated points. Unable to zoom or click on map markers.

Expected behavior:

User should be able to zoom, move around the map and click on map markers to see the building details.

### PR DESCRIPTION
**Pull request by vladplesu**
Closes #166 

- add here map api scripts to index.html

- add component for interactive map

- add component for map marker

- modify map container to use new interactive map

### CODE REVIEW

File: client/src/containers/home/MapFragment.js [hunk 1]
```diff
@@ -1,28 +1,81 @@
...
+    risk_category: 'RS II',
+    lat: 44.40553,
+    lng: 26.186,
+  },
+  {
+    general_id: 70,
+    risk_category: 'RS III',
+    lat: 44.41339,
+    lng: 26.146,
+  },
```
Comment (tudoramariei) [inline client/src/containers/home/MapFragment.js]: Could we connect this to [the mock JSON server](https://github.com/code4romania/seismic-risc/tree/master/client#use-mock-data-api) or would it be too big of an effort for this?
This should be solved by #92, but until then could we make use of this server so we wouldn't have static data? 🤔

File: client/src/containers/home/MapFragment.js [hunk 2]
```diff
@@ -1,28 +1,81 @@
...
 ];
 
 export default () => {
   return (
     <Tabs size="large" animated={false} defaultActiveKey="1" onChange={null}>
       <TabPane tab="Toate clădirile cu risc seismic" key="all">
-        <HereMap points={dummyPoints} />
+        {/* apikey to be set */}
```
Comment (tudoramariei) [inline client/src/containers/home/MapFragment.js]: I'm not sure what this is for. We're setting `apiKey=""` on the second line this so shouldn't we have a variable or something similar here?

File: client/src/containers/home/MapFragment.js [hunk 3]
```diff
@@ -1,28 +1,81 @@
...
+        <HereMapInteractive apikey="" points={dummyPoints} />
       </TabPane>
       <TabPane tab="Clasa U1 de risc seismic" key="classU1">
-        <HereMap points={dummyPoints} />
+        {/* apikey to be set */}
```
Comment (tudoramariei) [inline client/src/containers/home/MapFragment.js]: Should `apiKey` be set manually in all of these places? Can't we use a variable which we manually set and these `apiKey`s just use that? 
**I'm not saying to use a configuration file**, just set the key in one place in a `*.js` file and then have all these `apiKey` variables make use of it.

**Comment by vladplesu** [comment]
If this is ok now, I will merge the two commits in to a single one.

**Pull request merged**